helpers/announce.js

```diff
@@ -11,13 +20,22 @@ export function announce(message) {
 		container.style.clip = 'rect(0px,0px,0px,0px)';
 		document.body.appendChild(container);
 	}
-	/* VO has strange hueristics for announcement - it will not announce
-	duplicate message unless it is additive, but we prefer content to not grow */
-	if (message !== lastMessage) {
-		container.innerHTML = '';
-	}
-	requestAnimationFrame(() => {
+
+	/* Need to give browser enough time to create the live region so that it will
+	treat the change as an update. Firefox sometimes ignores changes if the region
+	and update are made too quickly in succession. RequestAnimationFrame is not
+	sufficient here. */
+	setTimeout(() => {
 		container.appendChild(document.createTextNode(message));
-		lastMessage = message;
-	});
+	}, 100);
```

Comment:
Coincidently, this is the exact same delay that iron-a11y-announcer used, although a slightly shorter delay may be possible.